modules/nf-core/gatk4/markduplicates/environment.yml

```diff
@@ -5,4 +5,4 @@ channels:
   - defaults
 dependencies:
   - bioconda::gatk4=4.4.0.0
-  - bioconda::samtools=1.17
+  - bioconda::samtools=1.18
```

Comment:
this one here should need a mulled bump then as well. I think either update the container or leave this one out

Reply:
Oh good spot, thanks I'll leave that one out!